### PR TITLE
Coach reports forms responsive hotfix

### PR DESCRIPTION
**Pull request by Antrikshy**
(Relatively) quick fix on #3120.

screen shot 2015-02-26 at 12 36 23 pm

I followed @arceduardvincent's idea of a horizontal form on wide screens. I also rearranged some classes and template structure for the other form on the page to make it consistent.

Not sure why some other commits tagged along when I merged develop into my branch locally.

### CODE REVIEW

**Comment by MCGallaspy** [comment]
Tests fail on `Downloaded assessmentitems from a remote server`...

File: kalite/coachreports/hbtemplates/coach_nav/reports-nav.handlebars [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
-            {{#ifcond selected "===" "test" }}selected{{/ifcond}}>
-                {{_ "Test Report" }}
-            </option>
-            <option value="spending"
-            {{#ifcond selected "===" "spending" }}selected{{/ifcond}}>
-                {{_ "Spending Report" }}
-            </option>
-            {{/if}}
-        </select>
+<form id="coachreport-select-form" class="form-vertical">
```
Comment (rtibbles) [inline kalite/coachreports/hbtemplates/coach_nav/reports-nav.handlebars]: Assuming this is the only relevant change in this code block?

**Comment by rtibbles** [comment]
Manually tested, does what it says on the tin. One outstanding issue with the tabular report section that I have opened as #3142 .

**Comment by rtibbles** [comment]
Tests pass. Merging.

**Comment by arceduardvincent** [comment]
I just notice that the spacing of the coach-report is has now bigger spacing both side.
For better view we must ditch the spacing both side.

I'll attach the image below can you have a take a look on it.
screen shot 2015-02-27 at 12 19 18 pm 2

**Pull request merged**